tutorial/transport/linopy_model.py

```diff
@@ -0,0 +1,126 @@
+# m = linopy.Model()
+# i = {"Canning Plants": ["seattle", "san-diego"]}
+# j = {"Markets": ["new-york", "chicago", "topeka"]}
+# a = xr.DataArray([350, 600], coords=i, name="capacity of plant i in cases")
+# b = xr.DataArray([325, 300, 275], coords=j, name="demand at market j in cases")
+# d = xr.DataArray(
+#     [[2.5, 1.7, 1.8], [2.5, 1.8, 1.4]],
+#     coords=i | j,
+#     name="distance in thousands of miles",
+# )
+# f = 90  # Freight in dollars per case per thousand miles
+# c = d * f / 1000
+# c.name = "transport cost in thousands of dollars per case"
+# x = m.add_variables(lower=0.0, coords=c.coords, name="Shipment quantities in cases")
+# con = x.sum(dim="Markets") <= a
+# con1 = m.add_constraints(con, name="Observe supply limit at plant i")
+# con = x.sum(dim="Canning Plants") >= b
+# con2 = m.add_constraints(con, name="Satisfy demand at market j")
+# obj = c * x
+# m.add_objective(obj)
+# print(linopy.available_solvers)
+# m.solve("highs")
+# print(x.solution)
+
+import linopy
+import pandas as pd
+
+from ixmp4.core import Equation, IndexSet, Parameter, Scalar
+from ixmp4.core import OptimizationVariable as Variable
+
+
+def create_set(indexset: IndexSet, name: str | None = None) -> pd.Index:
+    return pd.Index(indexset.elements, name=name or indexset.name)
+
+
+def create_parameter(
+    parameter: Parameter, index: pd.Index | list[pd.Index], name: str
+) -> pd.Series:
+    if isinstance(index, list):
+        index = pd.MultiIndex.from_product(index)
+
+    return pd.Series(data=parameter.values, index=index, name=name)
+
+
+def create_dantzig_model(
+    i: IndexSet,
+    j: IndexSet,
+    a: Parameter,
+    b: Parameter,
+    d: Parameter,
+    f: Scalar,
```

Comment:
Per my other comment in the notebook, this should be streamlined to only take the `run`. Looking ahead to a MESSAGE use case, we don't want to list all sets and parameters when calling something like
```
message_ix.run(Run)
```
(where `run()` is a shorthand for writing to gdx, calling GAMS to solve, reading the solution).
```suggestion
    run: Run,
```